demo/list/list-view.html

```diff
@@ -81,34 +87,42 @@
       },
 
       animationConfig: {
-        type: Object,
-        value: function() {
-          return {
-            'entry': [{
-              name: 'fade-in-animation',
-              node: this.$.button
-            }],
-
-            'exit': [{
-              name: 'fade-out-animation',
-              node: this.$.button
-            }, {
-              name: 'hero-animation',
-              id: 'hero',
-              fromPage: this
-            }]
-          };
-        }
+        type: Object
       }
+    },
 
+    attached: function() {
+      this.animationConfig = {
```

Comment:
this is different from 1.x, as at every `attached` you'd override `this.animationConfig`.
You could change this to `this.animationConfig = this.animationConfig || {`